### ISSUE TITLE
Fix r_primitives for AMD Radeon cards

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#803]
Author: @adanski (Dragonji)
Date: 2015-01-29
Redmine Issue: 803, https://dev.etlegacy.com/issues/803

---

For some reason performance is awful if @r_primitives@ is other than @2@ on post-13.9 Catalyst drivers.

http://forums.amd.com/game/messageview.cfm?catid=443&threadid=171189 (link dead)
http://web.archive.org/web/20150429193417/http://forums.amd.com/game/messageview.cfm?catid=443&threadid=171189

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2015-01-29T21:24:43Z

---

That's... an AMD/ATI driver bug, and not directly related to ET:L.

You might want to expand the FAQ and I believe we can close this ticket - there's nothing we can/should do here.

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2015-01-29T21:26:20Z

---

Oh, I see you've already done it. Closing now.

**Comment by ETLegacyTeam** [comment]
From: Anonymous (Anonymous)
Date: 2015-02-01T23:31:51Z

---

This effects NVIDIA cards too. I've never had to use this cvar in my 12 years of ET until about a month to three ago. I'm using a GTX 780 Ti and have to use this or my FPS will be in the 50s and 60s and unstable. As soon as I set /r_primitives to 2, my FPS becomes a solid 100 again. For some reason I just started my game and suddenly it had horrible FPS, I hadn't even updated the driver or changed anything. I went to bed the night before, woke up and when I came on the computer I went to play ET and the FPS was poor. 

Current NVIDIA Driver: 347.25 (also occured on last couple drivers)
Video card: http://www.evga.com/products/Specs/GPU.aspx?pn=88601331-dbce-436d-8eb3-37f1f63b6745
